src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -2397,6 +2397,20 @@ elementFormDefault="qualified">
             </xs:complexContent>
         </xs:complexType>
     </xs:element>
+    <xs:element name="GetFileHash" substitutionGroup="msb:Task">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="msb:TaskType">
+                    <xs:attribute name="Files" use="required" />
+                    <xs:attribute name="Algorithm" default="SHA256" />
+                    <xs:attribute name="MetadataName" default="FileHash" />
+                    <xs:attribute name="HashEncoding" default="hex" />
+                    <xs:attribute name="Hash" />
+                    <xs:attribute name="Items" />
```

Comment:
`Hash` and `Items` are outputs, so I don't think they should be listed here. But outputs are listed for other tasks. That confuses me. I guess leave it in.

Reply:
ok